com.usebottles.bottles.yml

```diff
@@ -42,7 +42,7 @@ add-extensions:
 
   org.freedesktop.Platform.codecs_extra.i386:
     directory: lib/i386-linux-gnu/codecs-extra
-    version: '25.08-extra'
+    version: '24.08-extra'
```

Comment:
There's no such version. It should be removed completely. In the hindsight merging https://github.com/flathub/com.usebottles.bottles/pull/506 would save you a lot of effort.